client/dashboard/sites/hosting-feature-activation-modal/data-center-form.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, Card, CardBody, SelectControl } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import InlineSupportLink from '../../components/inline-support-link';
+import { Text } from '../../components/text';
+
+const DataCenterOptions = [
+	{
+		value: '',
+		get label(): string {
```

Comment:
Whoa! Getters 😅 
Why not just
```ts
{
  value: '',
  label: __( 'Optimal data center' ),
}
```
?

Reply:
Carried over from existing code, updated!